include/gdf/cffi/functions.h

```diff
@@ -1,10 +1,14 @@
 /* column operations */
+#ifndef _GDF_FUNCTIONS_H_
```

Comment:
Files under the `gdf/cffi` directory is to be parsed by python's `cffi` module, which doesn't support any preprocessor.  The headers in the directory is already included in `gdf/gdf.h`, thus it is not expected to be included directly.  So, the `#ifndef` is not needed.

Reply:
ok we will drop that too